modules/auxiliary/voip/telisca_ips_lock_abuse.rb

```diff
@@ -0,0 +1,167 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  #include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Telisca IPSLock Abuse',
+      'Description'    => %q{This modules will exploit the vulnerabilities of Telisca IPSLock , in order to lock/unlock IP Phones.  you need to be in the voip vlan and you have to  know the phone name example : SEP002497AB1D4B .  Set ACTION to either LOCK or UNLOCK UNLOCK is the default.},
+      'References'     =>
+        [
+        ],
+      'Author'         =>
+        [
+          'Fakhir Karim Reda <karim.fakhir[at]gmail.com>',
+          'zirsalem'
+        ],
+       'DefaultOptions' =>
+        {
+          'SSL' => false,
+          'SSLVersion' => 'TLS1',
+          'RPORT' => 80
+        },
+       'License'        => MSF_LICENSE,
+       'DisclosureDate' => "Dec 17 2015",
+      'Actions'     =>
+        [
+          ['LOCK'],
+          ['UNLOCK']
+        ],
+    ))
+    register_options(
+      [
+        OptInt.new('TIMEOUT', [true, 'HTTP/HTTPS connect/read timeout in seconds', 1]),
```

Comment:
TIMEOUT can be removed. The HTTPClient mixin has the default value of 20 seconds. It available in the advanced datastore option so can be changed if required.